[HCCDOC#1739]: Document Insight Operator configmap config (#71527)
openshift/openshift-docs: modules/insights-operator-configuring-sca.adoc

[id="insights-operator-configuring-sca_{context}"]
= Configuring simple content access import interval

You can configure how often the Insights Operator imports the simple content access (sca) entitlements by using the *insights-config* ConfigMap in the `openshift-insights` namespace. The entitlement import normally occurs every eight hours, but you can shorten this sca interval if you update your simple content access configuration in the *insights-config* ConfigMap.

This procedure describes how to update the import interval to two hours.

.Prerequisites

* Remote health reporting is enabled, which is the default.
ifndef::openshift-rosa,openshift-dedicated[]
* You are logged in to the {product-title} web console as a user with the `cluster-admin` role.
endif::openshift-rosa,openshift-dedicated[]
ifdef::openshift-rosa,openshift-dedicated[]
* You are logged in to the {product-title} web console as a user with the `dedicated-admin` role.
endif::openshift-rosa,openshift-dedicated[]
* The *insights-config* ConfigMap exists in the `insights-openshift` namespace.

.Procedure

> . Go to *Workloads* -> *ConfigMaps* and select *Project: insights-openshift*.

Reviewer: Same here please.

Author: fixed

. Click on the *insights-config* ConfigMap to open it.
. Click *Actions* and select *Edit ConfigMap*.
. Click the *YAML view* radio button.
. In the file, set the *sca* attribute to `interval: 2h`; for example, to import content every two hours. You can also specify hours (h) or hours and minutes, for example: 2h30m.
. Click *Save*.

> . On the *insights-config* ConfigMap details page, scroll down to the Data section to view `config.yaml`. You can see that the *sca* attribute is set to `interval: 2h`.

Reviewer 2: ```suggestion
. On the *insights-config* ConfigMap details page, scroll down to the Data section to view `config.yaml`. Verify that the *sca* attribute is set to `interval: 2h`.
```
